nf_core/modules/lint/environment_yml.py

```diff
@@ -82,41 +99,86 @@ def environment_yml(module_lint_object: ComponentLint, module: NFCoreComponent,
             )
 
         if valid_env_yml:
-            # Check that the dependencies section is sorted alphabetically
-            def sort_recursively(obj):
-                """Simple recursive sort for nested structures."""
-                if isinstance(obj, list):
-
-                    def get_key(x):
-                        if isinstance(x, dict):
-                            # For dicts like {"pip": [...]}, use the key "pip"
-                            return (list(x.keys())[0], 1)
-                        else:
-                            # For strings like "pip=23.3.1", use "pip" and for bioconda::samtools=1.15.1, use "bioconda::samtools"
-                            return (str(x).split("=")[0], 0)
-
-                    return sorted([sort_recursively(item) for item in obj], key=get_key)
-                elif isinstance(obj, dict):
-                    return {k: sort_recursively(v) for k, v in obj.items()}
-                else:
-                    return obj
-
-            sorted_dependencies = sort_recursively(env_yml["dependencies"])
-
-            # Direct comparison of sorted vs original dependencies
-            if sorted_dependencies == env_yml["dependencies"]:
-                module.passed.append(
+            # Define channel priority order
+            channel_order = {
+                "conda-forge": 0,
```

Comment:
We don't require any channels anymore https://github.com/nf-core/modules/pull/8554